packs/data/spell-effects.db/spell-effect-swampcall-critical-failure.json

```diff
@@ -19,7 +19,25 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "selector": "speed",
+                "selector": "land-speed",
+                "type": "circumstance",
+                "value": -15
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "fly-speed",
+                "type": "circumstance",
+                "value": -15
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "climb-speed",
+                "type": "circumstance",
+                "value": -15
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "burrow-speed",
                 "type": "circumstance",
                 "value": -15
             }
```

Comment:
For this we have "all-speeds"

Reply:
Except the effect applies to all except swim